kokoro/unittests/presubmit.cfg

```diff
@@ -0,0 +1,7 @@
+# -*- protobuffer -*-
```

Comment:
Please also add copyright to this file. See [this](https://github.com/google/dwh-assessment-extraction-tool/blob/6d61bc74ef2e4bf040182fc37e3fa4a46fd0c557/kokoro/gcp_ubuntu/presubmit.cfg) as an example.